### PR TITLE
Feature/tdr 1/phpunit8 upgrade

### PR DESCRIPTION
**Pull request by Yury-Razhkou**
related to: https://oat-sa.atlassian.net/browse/TDR-1
depends on: https://github.com/oat-sa/generis/pull/751
Upgrade to phpunit8

### CODE REVIEW

File: test/unit/models/classes/LtiProvider/ConfigurableLtiProviderRepositoryTest.php [hunk 1]
```diff
@@ -101,7 +101,7 @@ public function testConstructorWithNullProviderListThrowsException()
         $subject = new ConfigurableLtiProviderRepository([
             ConfigurableLtiProviderRepository::OPTION_LTI_PROVIDER_LIST => null
         ]);
-        $this->setExpectedException(InvalidArgumentException::class);
+        $this->expectException(InvalidArgumentException::class);
```
Comment (bartlomiejmarszal) [inline test/unit/models/classes/LtiProvider/ConfigurableLtiProviderRepositoryTest.php]: I'm worried that this will be fixed in separate PR related to upgrading code to PHPUnit 7.

Comment (mike-ionut-mihai-sandu-tao) [reply test/unit/models/classes/LtiProvider/ConfigurableLtiProviderRepositoryTest.php]: i think we can drop the PR related to phpunit 7, because we are going to version 8
